compiler/rustc_mir_build/src/thir/pattern/check_match.rs

```diff
@@ -897,17 +897,16 @@ pub(crate) fn joined_uncovered_patterns<'p, 'tcx>(
     witnesses: &[DeconstructedPat<'p, 'tcx>],
 ) -> String {
     const LIMIT: usize = 3;
-    let pat_to_str = |pat: &DeconstructedPat<'p, 'tcx>| pat.to_pat(cx).to_string();
```

Comment:
can you revert this? make sure your temporary dirs are all clean (`target` and `build`, maybe even run `git clean -Xdf` if needed, beware of its destructive properties tho)

Reply:
reverted, still surprising to me that cleaning would cause different behavior than an incremental build.